account_menu_invoice_refund/views/account_invoice_view.xml

```diff
@@ -7,8 +7,8 @@
         <field name="res_model">account.move</field>
         <field name="view_mode">tree,form</field>
         <field eval="False" name="view_id" />
-        <field name="domain">[('type', 'in', ('out_invoice', 'out_refund'))]</field>
-        <field name="context">{'type':'out_invoice', 'journal_type': 'sale'}</field>
+        <field name="domain">[('move_type', 'in', ('out_invoice', 'out_refund'))]</field>
+        <field name="context">{'move_type':'out_invoice', 'journal_type': 'sale'}</field>
```

Comment:
change move_type to default_move_type. i think that journal_type is not required